package.json

```diff
@@ -219,8 +227,6 @@
     "tslint": "5.17.0",
     "tslint-config-prettier": "1.18.0",
     "tslint-defocus": "2.0.6",
-    "typedoc": "0.15.0",
-    "typedoc-plugin-no-inherit": "1.1.10",
```

Comment:
Typedoc is no longer needed as a direct dependency. May need later for clr-angular.